desc/geometry/surface.py

```diff
@@ -116,14 +124,24 @@ def __init__(
                 sym = False
 
         self._R_basis = DoubleFourierSeries(
-            M=self._M, N=self._N, NFP=NFP, sym="cos" if sym else False
+            M=MR,
```

Comment:
why change these from the self._M and self._N that they were previously?

Reply:
I don't remember making that change. Reverting back...